ref/ams-configuration.md

```diff
@@ -26,6 +26,16 @@ AMS provides various configuration items to customise its behaviour. The followi
 | `registry.url` | string | - | URL of the application registry to use |
 | `scheduler.strategy` | string | spread | Strategy the internal container scheduler in AMS is using to distribute container across available LXD nodes. Possible values are: `binpack`, `spread` |
 
+## Node-specific configuration
+
+In a cluster setup, there are configuration items that can be customised for each node. See [Configure cluster nodes](tbd) for instructions on how to set these configuration items.
+
+| Name | Type | Default |  Description            |
+|------|------|---------|-------------------------|
+| `cpu-allocation-rate` | integer | 4 | CPU allocation rate used for [over-committing resources](tbd#overcommitting) |
+| `gpu-slots` | integer | 0 | Number of [GPU slots](tbd#gpu-slots) available on the node |
+| `memory-allocation-rate` | integer | 2 | Memory allocation rate used for [over-committing resources](tbd#overcommitting) |
```

Comment:
Are those all available node-specific options? Or do we need to add others (or specify that these are "some of the configuration items")?

Reply:
No, there are more. See https://paste.ubuntu.com/p/H56J4HtGmQ/.
However after adding a node to the  AMS, modifying other options like `public-address`, `cpus`, `memory`, `unscheduable` makes less sense.
And we need to mention `gpu-encoder-slots` here.